src/GameDataReader/Common/Parsing/Setting.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace GameDataReader.Common.Parsing;
+
+internal abstract class Setting
```

Comment:
Abstract classes that don't have behavior should be interfaces (there are probably exceptions if you wanna pipe some abstraction layers but this is the last layer so we want it here)

(I pushed the changes for that already - also for all the other similar classes)